src/coniferest/calc_paths_sum.pyx

```diff
@@ -91,16 +95,15 @@ cdef void _paths_sum(selector_t [::1] selectors,
                      np.float64_t [::1] weights=None,
                      int num_threads=1):
 
-    cdef Py_ssize_t trees
+    cdef Py_ssize_t trees = indices.shape[0] - 1
     cdef Py_ssize_t tree_index
     cdef Py_ssize_t x_index
     cdef selector_t selector
     cdef Py_ssize_t tree_offset
     cdef np.int32_t feature, i
+    cdef int use_threads_if = (200 * num_threads < data.shape[0] * trees)
```

Comment:
Let's make it a variable to not have a "magical" constant

Reply:
Done